master/buildbot/worker/ec2.py

```diff
@@ -538,12 +538,29 @@ def _wait_for_instance(self):
             self.failed_to_start(self.instance.id, self.instance.state['Name'])
 
     def _wait_for_request(self, reservation):
```

Comment:
please rename to show that this method is called from within a thread (time.sleep is big no-no in normal async workflow)

```suggestion
    def _thd_wait_for_request(self, reservation):
```